### PR TITLE
doc: misc updates

### PR DESCRIPTION
**Pull request by annwoj**
small fixes

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]

File: doc/nrf/app_dev/board_support/board_names.rst [hunk 1]
```diff
@@ -11,10 +11,10 @@ Board names
 The following tables list all boards and corresponding board targets for Nordic Semiconductor's hardware platforms.
 
 The board targets follow Zephyr's :ref:`zephyr:board_terminology` scheme and are used mostly when :ref:`building`.
-For example, the board target ``nrf54l15pdk@0.2.1/nrf54l15/cpuapp`` can be read as made of the following elements:
+For example, the board target ``nrf54l15dk@nrf54l15/cpuapp`` can be read as made of the following elements:
```
Comment (nordicjm) [inline doc/nrf/app_dev/board_support/board_names.rst:14]: -@+/

**Review by nordicjm — CHANGES_REQUESTED** [review]

File: doc/nrf/app_dev/board_support/board_names.rst (hunk 1 above)
Comment (annwoj) [reply doc/nrf/app_dev/board_support/board_names.rst]: fixed

**Review by nordicjm — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Comment by rlubos** [comment]
@annwoj Should this be backported or is there no label intentionally?

**Comment by annwoj** [comment]
> @annwoj Should this be backported or is there no label intentionally?

Added label here as well

**Pull request merged**